src/webrtc.ts

```diff
@@ -613,7 +614,11 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
    * Stanza Handlers
    */
   private async handlePropose (msg: ProposeStanza) {
-    if (msg.from === this.jid) {
+    // if (msg.from === this.jid) {
+    //   return;
+    // }
+
+    if (msg.from in this.sessionsMap) {
```

Comment:
I might be missing something about what you were thinking with this check, but I think we should just using the existing check. `sessionsMap` is storing session IDs, not JIDs, right?

Reply:
Yeah but this sorta falls into my confusion before... I _thought_ this was an appropriate replacement for commenting out the block that I did but I'm more and more convinced we don't need to comment out that block to begin with